components/currency-dropdown.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+module.exports = React.createClass({
+  render: function() {
+    var currencies = this.props.currencies;
+    return (
+      <select onChange={this.props.onChange} className="currency-dropdown" value={this.props.currency}>
+        {Object.keys(currencies).map(function(currency, i) {
+          return (
+            <option value={currency} key={i}>
+              {currency + " " + currencies[currency].symbol}
```

Comment:
We also display symbol beside the other amount box. Is that ok there? Is it only the dropdown that is the issue?